test/UnitTests/Microsoft.OData.Core.Tests/UrlValidation/ODataUrlValidatorTests.cs

```diff
@@ -153,6 +153,55 @@ public static void CallingOnUriSucceeds()
             });
         }
 
+        [Fact]
+        public static void ValidatesQueryOptions()

```

Comment:
This test method name could be more descriptive.

Reply:
updated to `ValidatorValidatesCustomQueryOptions`, which is really what it tests (that the rule is called if a custom query option exists -- this is what did not happen previously.)